scrambles/src/test/java/net/gnehzr/tnoodle/scrambles/AlgorithmBuilderTest.java

```diff
@@ -26,14 +23,14 @@ void testRedundantMoves() throws LazyInstantiatorException, InvalidMoveException
         assertFalse(moves.contains("3Lw"));
         assertFalse(moves.contains("3Dw"));
 
-        Map<String, LazyInstantiator<Puzzle>> lazyScramblers = PuzzlePlugins.getScramblers();
+        Map<String, Puzzle> lazyScramblers = new HashMap<>();
+        // FIXME Map<String, Puzzle> lazyScramblers = PuzzlePlugins.getScramblers();
 
-        for (Map.Entry<String, LazyInstantiator<Puzzle>> lazyEntry : lazyScramblers.entrySet()) {
+        for (Map.Entry<String, Puzzle> lazyEntry : lazyScramblers.entrySet()) {
```

Comment:
This looks like a blocker: is this test testing anything now?

Reply:
Good catch! This `FIXME` really just slipped my intention because of the sheer size of this commit. I swear that this will be the last big infrastructure change for the forseeable future!!